sunpy/net/attr.py

```diff
@@ -194,7 +194,7 @@ def update_values(cls, adict):
                         if not name.isidentifier():
                             # This should account for names with one number first.
                             # We match for single digits at the start only.
-                            if _REGEX.match(name):
+                            if _REGEX.match(name) and len(name) > 1:
```

Comment:
We still want to do the replacement even if the length is 1, but we just don't want to add the `_`

Reply:
Well the length is 1 if its just a number

Reply:
No, the regex will match anything that starts with one and only one number even if it is just one number.